guests/python/Justfile

```diff
@@ -24,6 +19,7 @@ export PYO3_CROSS_LIB_DIR := DOWNLOADS_DIR / "python-sdk"
 PYTHON_TMP_VENV := DOWNLOADS_DIR / "venv"
 PYTHON_SITE_PACKAGES := PYO3_CROSS_LIB_DIR / "lib" / "python" + PYO3_CROSS_PYTHON_VERSION / "site-packages"
 
+# Not used in the script but the linker must know where to find the WASI sysroot libs.
 export WASI_SDK_LINK_PATH := DOWNLOADS_DIR / "wasi-sysroot" / "lib" / "wasm32-wasip2"
```

Comment:
I think we should re-design `configure_linker` instead. `download_wasi_sdk` knows where it downloaded the SDK to, so it should tell `configure_linker` that